components/src/preact/components/annotated-mutation.tsx

```diff
@@ -0,0 +1,48 @@
+import { Fragment, type FunctionComponent } from 'preact';
+
+import type { SequenceType } from '../../types';
+import type { Deletion, Substitution } from '../../utils/mutations';
+import { useMutationAnnotation } from '../MutationAnnotationsContext';
+import { InfoHeadline1, InfoHeadline2, InfoParagraph } from './info';
+import { ModalButton } from './modal';
+
+export type AnnotatedMutationProps = {
+    mutation: Substitution | Deletion;
+    sequenceType: SequenceType;
+};
+
+export const AnnotatedMutation: FunctionComponent<AnnotatedMutationProps> = ({ mutation, sequenceType }) => {
+    const mutationAnnotations = useMutationAnnotation(mutation.code, sequenceType);
+
+    if (mutationAnnotations === undefined || mutationAnnotations.length === 0) {
+        return mutation.code;
+    }
+
+    const modalContent = (
+        <div className='block'>
+            <InfoHeadline1>Mutation Annotations</InfoHeadline1>
```

Comment:
I think that would be useful:

```suggestion
            <InfoHeadline1>Annotations for {mutation.code}</InfoHeadline1>
```